ui/src/app/components/runs/runs.component.html

```diff
@@ -34,9 +34,17 @@
 
     <clr-dg-column [clrDgField]="dagRunColumns.WORKFLOW_NAME" [clrFilterValue]>
       <ng-container *clrDgHideableColumn="{hidden: false}">Workflow name</ng-container>
+      <clr-dg-filter [clrDgFilter]="workflowNameFilter">
+        <app-string-filter #workflowNameFilter [property]="dagRunColumns.WORKFLOW_NAME" [removeFiltersSubject]="removeFiltersSubject">
```

Comment:
There is some duplication with `[property]` because it's already defined in the parent `clr-dg-column`. One way to resolve it is adding
```
  constructor(@Host() private column: ClrDatagridColumn) { }
```
to the `StringFilterComponent`. Then you can obtain the name from the column. However, obviously it adds a tight coupling to the `ClrDatagridColumn`, but one could argue that a filter without a column makes no sense

Reply:
Fixed

Reply:
No, I've reverted it back. It is clearer this explicit way. It simplifies testing and I would like to avoid having references on parent elements from a child.